src/engine/controls/cuecontrol.cpp

```diff
@@ -136,6 +137,73 @@ CueControl::~CueControl() {
     qDeleteAll(m_hotcueControls);
 }
 
+mixxx::audio::FramePos CueControl::nextTrigger(bool reverse,
+        mixxx::audio::FramePos currentPosition,
+        mixxx::audio::FramePos* pTargetPosition,
+        mixxx::audio::FrameDiff_t requested_frame) {
```

Comment:
Can we find a better name for `requested_frame`  ? Is it something like `lookAheadFrames` or such?